MonoGame.Framework/Android/MonoGameAndroidGameView.cs

```diff
@@ -94,6 +94,11 @@ void ISurfaceHolderCallback.SurfaceChanged(ISurfaceHolder holder, Android.Graphi
                 }
             }
 
+            // When the game is resumed from a portrait orientation it may receive a portrait surface at first.
+            // If the game does not support portrait we should ignore it because we will receive the landscape surface a moment later.
+            if (width < height && (_game.graphicsDeviceManager.SupportedOrientations & DisplayOrientation.Portrait) == 0)

```

Comment:
I haven't managed to make that happen. Phones have a rotation lock to portrait, tablets have no rotation lock. Maybe with a tablet that has its homescreen locked to landscape.